### DISCUSSION TITLE
Ticket Update

### DISCUSSION THREAD

**Discussion by micah4232** [django-helpdesk/django-helpdesk#1100] [General]
I get this weird redirect function on submit either attachments or updating ticket w comments that it routes to my root domain for some reason/somehow. I dont know how this is even possible as its sitting on a replit build VM and there are no settings of forwarding yet. I do plan to deploy it to a sub domain like bk.rooturl.com or something but now it doesnt have this setup so no reason it should reddirect on submission. Was wondeirng if anyone else is seeing something like this.

PS I updated to the latest pypi as mentioned in the latest comments in the pull requets sections

**Comment by Benbb96** [comment]
Could check in your admin module if you created a [Site](https://docs.djangoproject.com/en/4.2/ref/contrib/sites/) and what's its domain URL ?

I believe this is linked to [this part](https://github.com/django-helpdesk/django-helpdesk/blob/67eb0974c7f163216ececc1d8a715a0144d0375c/helpdesk/models.py#L721) of the code.
